packages/react-scripts/template/README.md

```diff
@@ -1,21 +1,32 @@
-[![Build Status](https://travis-ci.org/{GITHUB_ORG}/{GITHUB_REPO}.svg?branch=master)](https://travis-ci.org/{GITHUB_ORG}/{GITHUB_REPO})
+[![Build Status](https://travis-ci.org/fs-webdev/{GITHUB_REPO}.svg?branch=master)](https://travis-ci.org/fs-webdev/{GITHUB_REPO})
+
+This project was bootstrapped with FamilySearch's [Fork of Create React App](https://github.com/fs-webdev/create-react-app).
+
+## Running App Locally
+There are a few steps to get this app cloned and ready to run locally.
+1. Clone this repo to your machine  
+  `git clone https://github.com/fs-webdev/{GITHUB_REPO}.git`
+2. Run `npm install`
+3. Make sure you have @fs/fr-cli installed  
+  `npm i -g @fs/fr-cli`
+4. Run `fr env` to make a .env file for you automatically
+5. You should now be in a good state to run/develop the app locally
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 ## Available Scripts
 
 In the project directory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Runs the app in the development mode on `localhost:{SOME_PORT}`. `SOME_PORT` gets set to whatever is in
+your .env file, or falls back to port 3000 if .env's PORT is not set.
```

Comment:
```suggestion
your .env file, or falls back to port 5000 if .env's PORT is not set.
```
don't we override this to 5000 somewhere?

Reply:
We do not. The 5000 that we see so often is from the PORT in the .env file. If you remove that, 3000 is the default.